bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -134,6 +134,39 @@ void DeleteInstance(bigtable::InstanceAdmin instance_admin, int argc,
 }
 //! [delete instance]
 
+//! [create cluster]
+void CreateCluster(bigtable::InstanceAdmin instance_admin, int argc,
+                   char* argv[]) {
+  if (argc != 2) {
+    throw Usage{"create-cluster: <instance-id> <cluster-id>"};
+  }
+  std::string const instance_id = ConsumeArg(argc, argv);
+  std::string const new_cluster_id = ConsumeArg(argc, argv);
+  auto cluster_config =
+      bigtable::ClusterConfig("us-central1-f", 0, bigtable::ClusterConfig::HDD);
+  bigtable::DisplayName display_name("Put description here");
+  std::string cluster_id = instance_id + "-c1";
+  bigtable::InstanceConfig config(bigtable::InstanceId(instance_id),
+                                  display_name, {{cluster_id, cluster_config}});
+  config.set_type(bigtable::InstanceConfig::DEVELOPMENT);
+  auto instance = instance_admin.CreateInstance(config);
```

Comment:
I think `instance` is a future, if you call `CreateCluster()` immediately it might fail because the instance has not been fully created.  You should block on `instance.get()`.

Reply:
Also, you could rely on the user having already created the instance.  We do that in some of the other samples.  I should have thought about that in my previous review.

Reply:
So I think, no need to create Instance, just I have to use Instance_id passed by user..... right?

Reply:
> So I think, no need to create Instance, just I have to use Instance_id passed by user..... right?

Yes.  We will write a document or something that explains that you need to create one instance first, and then create the second cluster on it.